internal/controller/control-plane/helper_test.go

```diff
@@ -112,6 +112,9 @@ func expectModuleTemplateSpecGetReset(
 	if err != nil {
 		return err
 	}
+	if moduleTemplate.Spec.Data == nil {
+		return ErrExpectedLabelNotReset
```

Comment:
Looks weird. Is that a correct error to return?

Reply:
good catch, should be `ErrManifestResourceIsNil`